contracts/scripts/deploy.sh

```diff
@@ -14,6 +42,7 @@ function instrument_game_token {
 
   cd "$CONTRACTS_PATH"/$contract_name
 
+  # TODO : remove balance when token is mintable
```

Comment:
Should have an issue for that rather than a comment.

Reply:
I don't buy that no TODOs thing: I'd rather have a TODO in active development than forget

Reply:
I guess on a separate branch it is OK, but I have seen dead `TODO`s on main, so these things have a way of seeping in.